tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "forceConsistentCasingInFileNames": true,
     "lib": ["ESNext"],
     "module": "ESNext",
-    "moduleResolution": "Node",
+    "moduleResolution": "NodeNext",
```

Comment:
This is the key here—using `NodeNext` exposes all the problems with ESM your app may have. Caught several other bugs in the process.